utils/utils.py

```diff
@@ -32,7 +33,8 @@ def check_sender(m):
 
 
 # Global method for adding user to share and overseerr
-def add_user(client, user_email, user_name, overseerr_enabled: bool = False, overseerr_api: str = None, overseerr_server: str = None):
+def add_user(client, user_email, user_name, overseerr_enabled: bool = False, overseerr_api: str = None,
+             overseerr_server: str = None):
```

Comment:
I do not appreciate you wrapping my code to fit the PEP8 standard. Buy an ultrawide monitor. L + Ratio

Reply:
follow pep8 ty